target/linux/octeon/base-files/lib/preinit/79_move_config

```diff
@@ -10,6 +10,11 @@ move_config() {
 			[ -f "/mnt/$BACKUP_FILE" ] && mv -f "/mnt/$BACKUP_FILE" /
 			umount /mnt
 			;;
+		itus_shield-router)
+			mount -t vfat /dev/mmcblk1p1 /mnt
+			[ -f "/mnt/$BACKUP_FILE-$(board_name)" ] && mv -f "/mnt/$BACKUP_FILE" /
```

Comment:
that's different paths for check and move command. Do you need the board_name suffix here?